.github/workflows/release-discord-notify.yml

```
@@ -0,0 +1,27 @@
+name: release-discord-notify
```

Please name the workflow (and the file) as follows (by convention, and for scalability reasons):

```suggestion
name: notify-release
```